libs/audit-service/src/test/java/uk/gov/di/ipv/core/library/auditing/extension/AuditExtensionGpg45ProfileMatchedTest.java

```diff
@@ -79,6 +79,8 @@ void shouldSerializeToTheCorrectJson() throws Exception {
                         + "\"vcTxnIds\":[\"txn1\",\"txn2\",\"txn3\"]"
                         + "},"
                         + "\"timestamp\":1666170506"
+                        + ","
+                        + "\"event_timestamp_ms\":1666170506000"
```

Comment:
I'm wondering if it's worth updating that spuClock mock to return an instant that isn't an exact second? That way you could have this test value end in something other than `000` to give confidence it's actually using milliseconds, and not just multiplying some value for seconds by 1000.